mock/src/main/java/com/tngtech/keycloakmock/api/TokenConfig.java

```diff
@@ -27,7 +27,7 @@
  * <p>Example usage:
  *
  * <pre>{@code
- * TokenConfig config = TokenConfig.aTokenConfig().withSubject("subject).build();
```

Comment:
was this not broken?

Reply:
I'm not quite sure what you mean. If you're talking about the missing quotation mark: yes, that was broken.